gcsa/attachment.py

```diff
@@ -40,8 +40,7 @@ def __init__(self, file_url, title=None, mime_type=None, _icon_link=None, _file_
         .. _`available MIME types`: https://developers.google.com/drive/api/v3/mime-types
         """
 
-        if mime_type and mime_type not in Attachment._SUPPORTED_MIME_TYPES:
-            raise ValueError("Mime type {} is not supported.".format(mime_type))
+        self.unsupported_mime_type = mime_type not in Attachment._SUPPORTED_MIME_TYPES
```

Comment:
I think it's unnecessary to have this field. If you think it is a common use case to check for it, we can have

```python
@property
def supported_mime_type(self):
    return self.mime_type in Attachment._SUPPORTED_MIME_TYPES
```

I think it's better to avoid fields that are not present in Google's API to avoid confusion.